fmriprep/data/boilerplate.bib

```diff
@@ -220,6 +220,20 @@ @article{power_fd_dvars
     year = 2014
 }
 
+@article{satterthwaite,
```

Comment:
please give this an id more descriptive of the step, maybe:

```suggestion
@article{confounds,
```
or, if you want to keep the name it is fine with me:

```suggestion
@article{confounds_satterthwaite_2012,
```

(maybe we should move all citations to this style).